suggestions/audio_recorders.json

```diff
@@ -4,5 +4,17 @@
         "label": "Audio Recorder",
         "source": "fg",
         "repo": "https://github.com/Dimowner/AudioRecorder"
+    },
+    {
+        "id": "com.bnyro.recorder",
+        "label": "Record You",
+        "source": "f",
+        "repo": "https://github.com/Bnyro/RecordYou"
+    },
+    {
+        "id": "com.simplemobiletools.voicerecorder",
+        "label": "Simple Voice Recorder",
+        "source": "fg",
+        "repo": "https://github.com/SimpleMobileTools/Simple-Voice-Recorder"
```

Comment:
From what I used, Record You is far better than the Audio Recorder app that you suggested

Reply:
## Comparison between Audio Recorder and Record You

|Criterias|Audio Recorder|Record You| 
|--|--|--|
|Record audio| ✅ | ✅ |
|Record options | Naming pattern, audio format (m4a/wav/3gp), sample rate, bitrate, channel count | **Save folder location**, audio format (m4a/wav/3gp/**aac**/**opus**), sample rate, bitrate, channel count, **audio source**, naming pattern |
| App launcher shortcuts | ✅ | ✅ |
| Rename before saving | ✅ | ❌ |
| Recordings list | ✅ | ✅ |
| "Keep screen on while recording" feature | ✅ | ❌ |
|**Includes a screen recorder**| ❌ | ✅ |
| Screen record annotation tool | - | ✅ |
| Embedded audio stream to screen record | - | ✅ |
| **Theming** | Color palettes | Material You + light/dark |
| Waveform while recording | ✅ | ✅ |

Record You gains more points than Audio Recorder!

Reply:
> Record You gains more points than Audio Recorder!

It doesn't. This category is particularly for audio recorders, not screen recorders. This is a bonus feature but not a required one. Audio Recorder looks more polished and mature than Record You because the features it offers are very specific. For example, Record You has an option to set bitrate but there are no presets which is important from a user's point of view. Audio Recorder has two different ways to store the recordings: public and private spaces, and it goes in great length to ensure that they remain partitioned. This is a great privacy feature. It also offers a trash which I've seen in some stock recorders. It is more accessible than Record You (try using Talkback with Record You and Audio Recorder). Finally, it provides in-app documentation to assist the user.

About setting location, Record You allows you to set a custom location, but nowhere in the app does it show where the current recordings are being saved (even when you set one). Audio Recorder, on the other hand, stores files (when enabled) in a fixed directory that is understood by Android and most other recorders.

Record You is a new app. So, it requires some time before it can replace Audio Recorder (provided the developer do things right).

Reply:
I should file an issue about Record You instead of adding it to the suggestions list, just like the Filen app... 🤔

**Will be removed in the next commits and be moved to the issues tab!** ✅

Reply:
> I should file an issue about Record You instead of adding it to the suggestions list, just like the Filen app... thinking

Created: #32